bigchaindb/exceptions.py

```diff
@@ -1,65 +0,0 @@
-"""Custom exceptions used in the `bigchaindb` package.
```

Comment:
Any reason to not leave these exceptions in `bigchaindb`? Does `bigchaindb_common` use these exceptions?